src/types/aws.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+export type AWSError = {
+  +code: string, // a unique short code representing the error that was emitted.
+  +message: string, // a longer human readable error message
+  +retryable?: boolean, //whether the error message is retryable.
```

Comment:
nit:
```suggestion
  +retryable?: boolean, // whether the error message is retryable.
```
Also, what do you mean when you say `error message is retryable`? Do you mean the request?

Reply:
yes, retrying request when error occurred. 
I just have copied-pasted descriptions from the Property Details sections. https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Response.html